dotnet/src/InternalUtilities/src/Schema/KernelJsonSchemaBuilder.cs

```diff
@@ -22,16 +24,20 @@ namespace Microsoft.SemanticKernel;
 internal static class KernelJsonSchemaBuilder
 {
     private static JsonSerializerOptions? s_options;
-    private static readonly JsonSchemaMapperConfiguration s_config = new()
+    private static readonly AIJsonSchemaCreateOptions s_schemaOptions = new()
     {
-        IncludeSchemaVersion = false,
-        IncludeTypeInEnums = true,
-        TreatNullObliviousAsNonNullable = true,
+        IncludeSchemaKeyword = false,
```

Comment:
this should probably also:
- remove minimum with integer type, or keep minimum, but convert integer type to number + format
- remove uniqueItems on array type
See https://github.com/microsoft/semantic-kernel/issues/9786#issuecomment-2494441642 for more context

Reply:
collecting notes from the meeting: this PR does not change behaviour, it's simply a refactoring. Cleaning additional keywords would introduce a behaviour change.
This cleanup could also happen:
- OAI.net visitor pattern
- adding a new visitor pattern for the parser and object model in SK.
- pushing additional behaviour in AI libs, and consuming it here

We should most likely open a separate issue to discuss options, gage demand, and avoid blocking this PR here.

Reply:
> remove minimum with integer type, or keep minimum, but convert integer type to number + format

I don't believe either the M.E.AI schema generator or the original implementation that it replaces use the `minimum` or `maximum` keywords with integer types. At some point they were doing so for the `char` type specifically (using the schema `{ "type" : "string", "minimum" : 1, "maximum": 1 }` but that has since been removed following issues it was creating.

Reply:
None of these keywords appear in the generated schemas.